.github/workflows/build_osgeo4w.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/sh
+#
+# Usage: build_osgeo4w.sh [-p] [path]
+#
+# By default, the script will look for the source code in the current directory
+# and create bin.x86_64-w64-mingw32\grass$ver.bat (run this batch file to start
+# GRASS GIS) and dist.x86_64-w64-mingw32\etc\env.bat.
+#
+# -p	optionally install GRASS GIS to C:\OSGeo4W64\opt\grass (run
```

Comment:
are the binaries downloadable somewhere? could these be used to upload for daily builds in OSGeo4W?

Reply:
Not at this point, but we could use GitHub artifacts to copy out the package: https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

Reply:
ok. 

if we're going this way later on, maybe the install scheme (now in \opt\) have to be adapted to be usable in OSGeo4W?

Reply:
The current workflow is mainly for testing, but changing the install scheme wouldn't be a problem. I thought about that possibility too, but we already have working daily builds for Windows. Also, I wanted to create a portable package that doesn't require admin rights to install.